Dockerfile

```diff
@@ -1,18 +1,35 @@
-FROM quay.io/app-sre/fluentd-upstream:v1.16-1
+FROM registry.redhat.io/openshift-logging/fluentd-rhel9@sha256:a31038fa2b953660e09b7db8411e808075c9037b679ba56a4a6245f56853db56
```

Comment:
do we know which fluentd version is from this base? Or can we use https://catalog.redhat.com/software/containers/ubi9/ruby-33/657b07afefad7c69e591d468 and install fluentd via gem https://docs.fluentd.org/installation/install-by-gem?

Reply:
Currently fluentd is 1.16.1, the changes I have currently will make fluentd 1.16.2